Projects/hCoreUI/Sources/MarkdownTextView.swift

```diff
@@ -98,10 +74,10 @@ class CustomTextView: UIView, UITextViewDelegate {
 
     func setContent(from text: String) {
         configureTextView()
-        let schema = ColorScheme(UITraitCollection.current.userInterfaceStyle) ?? .light
+        let schema = ColorScheme.init(UITraitCollection.current.userInterfaceStyle) ?? .light
         let markdownParser = MarkdownParser(
-            font: Fonts.fontFor(style: fontStyle),
-            color: hTextColor.secondary.colorFor(schema, .base).color.uiColor()
+            font: Fonts.fontFor(style: config.fontStyle),
```

Comment:
Should be 18px always, changed this before. Have discussed this with William.

Reply:
Lets keep it like this, on this way we can use it everywhere. I have placed everywhere else to 18, but for chat banner it is 14.